test/puppetlabs/pcp/messaging_test.clj

```diff
@@ -162,7 +162,7 @@
                {:ssl-key "./test-resources/ssl/private_keys/client01.example.com.pem"
                 :ssl-cert "./test-resources/ssl/certs/client01.example.com.pem"})
     (eventually-logged? "puppetlabs.pcp.client" :warn
-                        (partial re-find #"TLS Handshake failed\. Sleeping for up to 200 ms to retry")
+                        (partial re-find #"TLS Handshake failed\. Sleeping for up to 400 ms to retry")
```

Comment:
Can we just make it `\d+` to avoid the race entirely? Although that could run afoul of the fact that durations > 1000ms include a comma. We could always fix _that_ too... yakkety yak...